### DISCUSSION TITLE
事件数据相关问题

### DISCUSSION THREAD

**Discussion by jzh1qaz** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#9] [Q&A] [answered]
作者你好，请问事件数据[events.json]的来源是什么呢？我在对比gamewith的事件数据时发现有些事件貌似有些错误（比如事件id：501025704），所以想要知道来源或者做数据的方式并尝试自己修改一下。

**Comment by EtherealAO** [comment, marked as answer]
主要数据来自kamigame，使用乌拉拉大胜利的数据获取事件与来源的关系，生成工具可能会在近几天开源（因为代码实在太烂了）

**Reply by EtherealAO** [reply]
https://github.com/EtherealAO/UmamusumeDeserializeDB5
